### DISCUSSION TITLE
Keeping the scroll position of the left TOC

### DISCUSSION THREAD

**Discussion by arnonzooz** [google/docsy#585] [Ideas]
Hi all,

I figured it's nice to have the left TOC scroll to the correct position when, for example, when clicking a link to a topic or when sending someone a link to a specific topic. The following two lines of code do the job:

```
const element = document.getElementById(window.location.pathname);
element.scrollIntoView({ behavior: 'smooth', block: 'nearest', inline: 'start' })
```

What do you think?

**Comment by narrenfrei** [comment]
There already exists a issue for this topic #348 with a similar solution from me. I wasn't able to get it ready, because there where some other things on my personal Docsy roadmap. But of cause, it's great when you get this ready.
